nzedb/NZBImport.php

```diff
@@ -297,15 +297,25 @@ protected function scanNZBFile(&$nzbXML, $useNzbName = false)
 			// Get the group names, groups_id, check if it's blacklisted.
 			$groupArr = [];
 			foreach ($file->groups->group as $group) {
-				$group = (string)$group;
+				$group = strtolower($group);
 
 				// If groups_id is -1 try to get a groups_id.
+				// why ask if $groupID === -1 when we did not change it since it was set it in line 281?
 				if ($groupID === -1) {
```

Comment:
Probably because we're in a loop and the value may (should) be changed afterwards.
However it does feel wrong somehow, I'll look at it closer.

Reply:
doh, maybe you're true, sometimes you don't see wood in the forest!